src/components/StoreInfo/StoreSocialTab.tsx

```diff
@@ -0,0 +1,146 @@
+
+export interface Props {
+    seller: BrowsePageSeller;
+}
+
+const StoreSocial: React.SFC<Props> = ({ seller }) => {
+    const [show, setShow] = useState(false);
+    const target = useRef(null);
+
+    const location = window.location.href;
+    const facebookQuote = 'Help raise money for ' + seller.name;
+    const socialIconForegroundColor = '#a9182e';
+    const socialIconBackgroundColor = 'white';
+    const socialIconDimensions = { height: 50, width: 50 };
+
+    const clipboardStyle = {
+        fill: '#a9182e',
+        backgroundColor: 'white',
+        display: 'inline-block',
+        verticalAlign: 'middle',
+        margin: '6px 8px 10px 14px'
+    };
+
+    return (
+        <React.Fragment>
+            <HeaderText>
+                Share this merchant
+            </HeaderText>
+
+            <SocialContainer>
+                <FacebookShareButton
+                    url={location}
+                    quote={facebookQuote}
+                    className="share"
+                >
+                    <SocialIcon
+                        network="facebook"
+                        fgColor={socialIconForegroundColor}
+                        bgColor={socialIconBackgroundColor}
+                        style={socialIconDimensions}
+                    />
+                    <SocialTextLabel>FACEBOOK</SocialTextLabel>
+                </FacebookShareButton>
+            </SocialContainer>
+
+            <SocialContainer>
+                <TwitterShareButton url={location} className="share">
+                    <SocialIcon
+                        network="twitter"
+                        fgColor={socialIconForegroundColor}
+                        bgColor={socialIconBackgroundColor}
+                        style={socialIconDimensions}
+                    />
+                    <SocialTextLabel>TWITTER</SocialTextLabel>
+                </TwitterShareButton>
+            </SocialContainer>
+
+            <SocialContainer>
+                <EmailShareButton url={location} className="share">
+                    <SocialIcon
+                        network="email"
+                        fgColor={socialIconForegroundColor}
+                        bgColor={socialIconBackgroundColor}
+                        style={socialIconDimensions}
+                    />
+                    <SocialTextLabel>EMAIL</SocialTextLabel>
+                </EmailShareButton>
+            </SocialContainer>
+
+            <SocialContainer>
+                <ClipboardContainer ref={target}>
+                    <CopyToClipboard text={location} onCopy={() => setShow(!show)}>
+                        <ClipboardIcon
+                            size={28}
+                            style={clipboardStyle}
+                        />
+                    </CopyToClipboard>
+                    <CopyToClipboard text={location} onCopy={() => setShow(!show)}>
+                        <SocialTextLabel>COPY TO CLIPBOARD</SocialTextLabel>
```

Comment:
Should we localize?

Reply:
Yes definitely! Will put out a followup PR for that

Reply:
https://trello.com/c/hw16UFsN/506-add-localization-to-merchant-tabs